source/MaterialXTest/Types.cpp

```diff
@@ -98,4 +106,84 @@ TEST_CASE("Matrices", "[types]")
     REQUIRE(quot2 == scale);
     REQUIRE(quot3 == trans);
     REQUIRE(quot4 == mx::Matrix44::IDENTITY);
+
+    // Matrix translation
+    mx::Vector3 amount44{ 1.0f, 2.0f, 3.0f };
```

Comment:
I would recommend the usual parentheses here instead of universal initialization brackets, to clarify to readers that the standard syntax works for vectors and matrices.  (I unnecessarily used universal initialization in the identity constants above, but that's something I'd like to fix in an upcoming edit...)